### PR TITLE
II-6523 Update insightagent-boilerplate about optimize performance and fuse metrics

### PR DESCRIPTION
**Pull request by zhangzinan**

### CODE REVIEW

File: kafka2/README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ python getmessages_kafka2.py -t
 
 ###### If satisfied with the output, configure the agent to run continuously:
 ```bash
-sudo ./setup/cron-config.sh
+sudo ./setup/monit-config.sh
```
Comment (yqian10) [inline kafka2/README.md:67]: from Jeremy this morning, should this be set back to cron?

File: kafka2/setup/monit-config.sh [hunk 2]
```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
```
Comment (yqian10) [inline kafka2/setup/monit-config.sh:1]: as you have base branch 6413 to this one, please keep code being updated with base branch

File: kafka2/getmessages_kafka2.py [hunk 3]
```diff
@@ -887,7 +914,7 @@ def parse_json_message(messages):
 
 
 def parse_json_message_single(message):
-    message = json.loads(json.dumps(message))
+    # message = json.loads(json.dumps(message))
```
Comment (yqian10) [inline kafka2/getmessages_kafka2.py:917]: why need to comment out message = json.loads(json.dumps(message))

Comment (zhangzinan) [reply kafka2/getmessages_kafka2.py:917]: this is no need and take more time.

**Pull request merged**